### PR TITLE
How to use embedding doc

### PR DESCRIPTION
**Pull request by jetfuel**
Add the 'How to use embedding' documentation.

### CODE REVIEW

File: demo/pytorch/embedding_tutorial_ch.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# 如何用VisualDL可视化 Embedding
+
+在这里，我们将向您展示如何在 PyTourch 中使用 VisualDL 可视化 Embedding。
```
Comment (daming-lu) [inline demo/pytorch/embedding_tutorial_ch.md]: PyTorch

File: demo/pytorch/embedding_tutorial_ch.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+Embedding 常用于自然语言处理中，他能将语义意义使用高维向量来表示。
```
Comment (daming-lu) [inline demo/pytorch/embedding_tutorial_ch.md]: 它， also '语义意义' means what? semantics?  If it is semantics, then just 语义 would work.

File: demo/pytorch/embedding_tutorial_en.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+# The first parameter is embedding list. The type is list[list[float]]
+# The second parameter is word_dict. The type is dictionary<string, int>.
+embedding.add_embeddings_with_word_dict(embeddings_list, word_to_ix)
+```
+
+Include the above code to your embedding training program.
+
+This will save the embeddings and the word dictionary to the `./embedding_log` folder.
+
+We can now activate the VisualDL by running `visualdl --logdir=./embedding_log`.
```
Comment (nickyfantasy) [inline demo/pytorch/embedding_tutorial_en.md]: use visualDL as case sensitive?

**Review by nickyfantasy — DISMISSED** [review]
LGTM

File: demo/pytorch/embedding_tutorial_en.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
+# How to visualize embedding with VisualDL
+
+Here we would like to show you how to visualize embeddings with
+VisualDL in PyTourch.
```
Comment (daming-lu) [inline demo/pytorch/embedding_tutorial_en.md]: PyTorch

File: demo/pytorch/embedding_tutorial_en.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+        # Get the Python number from a 1-element Tensor by calling tensor.item()
+        total_loss += loss.item()
+    losses.append(total_loss)
+print(losses)  # The loss decreased every iteration over the training data!
+```
+
+That's all the code you need to generate your first embedding.
+
+Now, let just add a little bit of code to store the embedding to VisualDL log
```
Comment (daming-lu) [inline demo/pytorch/embedding_tutorial_en.md]: let us

File: demo/pytorch/embedding_tutorial_en.md (hunk 3 above)
Comment (jetfuel) [reply demo/pytorch/embedding_tutorial_en.md]: Sounds good. Changed

File: demo/pytorch/embedding_tutorial_en.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+
+embeddings_list = model.embeddings.weight.data.numpy()  # convert to numpy array
+
+# VisualDL embedding log writer takes two parameters
+# The first parameter is embedding list. The type is list[list[float]]
+# The second parameter is word_dict. The type is dictionary<string, int>.
+embedding.add_embeddings_with_word_dict(embeddings_list, word_to_ix)
+```
+
+Include the above code to your embedding training program.
```
Comment (daming-lu) [inline demo/pytorch/embedding_tutorial_en.md]: Insert the above code snippet into ...

File: demo/pytorch/embedding_tutorial_ch.md (hunk 2 above)
Comment (jetfuel) [reply demo/pytorch/embedding_tutorial_ch.md]: got it

**Review by daming-lu — APPROVED** [review]
